guide/daily-spending-wallet/settings.md

```diff
@@ -0,0 +1,213 @@
+    height = 600
+    layout = "full-width"
+%}
+
+<!--
+
+
+This is more of a summary page that bring together all the application settings that
+the user encounters throughout the app experience into a single, concise form.
+
+
+-->
+
+# Settings
+{: .no_toc }
+
+---
+
+<div class="glossary-toc" markdown="1">
+ * Table of contents
+{:toc}
+</div>
+
+---
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/settings/overview.png"
+   retina = "/assets/images/guide/daily-spending-wallet/settings/overview@2x.png"
+   modalImage = "/assets/images/guide/daily-spending-wallet/settings/overview-big.png"
+   alt-text = ""
+   width = 250
+   height = 541
+   layout = "float-right-desktop -background -shadow"
+   caption = "The overview screen provides an easy-to-understand grouping of settings."
+%}
+
+As you probably noticed in previous pages, various settings can be configured contextually throughout the app experience (like [hiding and unhiding the balance]({{ '/guide/daily-spending-wallet/privacy/#hiding-sensitive-information' | relative_url }})). All these options, and a few extra ones, are bundled in the settings area.
+
+While not the most frequently visited part of an application, it fills an important functional role by giving users precise control over their experience.
+
+</div>
+
+## General
+
+This sub-section includes various non-essential settings that are not covered in the other categories and let users adjust the application to their context and personal preferences.
+
+{% include image-gallery.html pages = page.images_general %}
+
+## Fees
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/settings/fees.png"
+   retina = "/assets/images/guide/daily-spending-wallet/settings/fees@2x.png"
+   modalImage = "/assets/images/guide/daily-spending-wallet/settings/fees-big.png"
+   alt-text = ""
+   width = 250
+   height = 541
+   layout = "float-right-desktop -background -shadow"
+   caption = "The fees screen provides a clear overview of all fees that may be charged."
+%}
+
+Paying fees for transactions is unavoidable and typically not something users appreciate. Being transparent and communicating them clearly in context (e.g. when [reviewing a payment]({{ '/guide/daily-spending-wallet/sending/#review--approval' | relative_url }}) before sending it) is crucial. Since contextual information is selective, this fee screen in settings is needed to provide a concise summary of all the types of fees that may be charged.
+
+During regular use, only lightning network fees will apply to transactions in this wallet. An additional fee may be charged for opening new channels to increase liquidity.
+
+When users send to an on-chain address, a swap fee applies. This fee also applies if the user receives to an on-chain (swap) address. This may cause confusion, as typically the sender is reponsible for paying fees.
```

Comment:
```suggestion
When users send to an on-chain address, a swap fee applies. This fee also applies if the user receives to an on-chain (swap) address. This may cause confusion, as typically, the sender is responsible for paying fees.
```

Reply:
This also only occurs if the business charges for that fee otherwise it's just a normal on-chain transascton fee.

May want to add a sentence or two about fees that wallet providers charge (non-network ones) and how these should also be clearly communicated.

Reply:
How would you word this?